djangomodelimport/utils.py

```diff
@@ -0,0 +1,20 @@
+import dataclasses
+from typing import runtime_checkable, Protocol, Iterable
+
+from django.forms import Field
+
+
+@runtime_checkable
+class HasSource(Protocol):
+    source: str | Iterable[str]
+
+
+@dataclasses.dataclass
+class ImportHeader:
+    field_name: str
+    field: Field
+    name: str
+    required: bool
+    display: str = ""
+    help_text: str = ""
+    alternatives: list[list["ImportHeader"]] = dataclasses.field(default_factory=list)
```

Comment:
Is it worth adding a couple of comments to these? Particularly `name` vs `field_name`, and `alternatives`.